libraries/c_sdk/standard/mqtt/src/iot_mqtt_subscription.c

```diff
@@ -527,7 +524,7 @@ void _IotMqtt_RemoveSubscriptionByPacket( _mqttConnection_t * pMqttConnection,
         .order            = order
     };
     int8_t contextIndex = -1;
-    bool mutexStatus = true, subscriptionStatus = true;
+    bool mutexStatus = true;
```

Comment:
Suggest rewriting the logic in lines 532-543 as follows to improve clarity:

```c
mutexStatus = IoTMutex_Take( &( connToContext[ contextIndex ].subscriptionMutex ) );

if( mutexStatus == true )
{
    /* lines 534-537 are unchanged from the original implementation */
}
/* lines 539-543 from the original implementation disappear */
```

Same comment for lines 647 - 674.

Reply:
Thanks. Addressed it.